package-lock.json

```diff
@@ -1,17 +1,17 @@
 {
   "name": "leaflet-environmental-layers",
```

Comment:
Some of the changes in this file are downgrades... Would you mind removing it from this PR? Thanks!

Reply:
Sorry I forgot to remove this file